docs/migrating-to-tool-kit.md

```diff
@@ -227,18 +229,39 @@ hook for a `test` target, for example,) but sometimes you'll want to leave some
 of the targets in at the beginning of the migration and gradually transition
 everything over to Tool Kit. Regardless, this step just provides some
 suggestions and the migration tool doesn't perform any actions so you'll need to
-delete the Makefile yourself if you don't think it's needed anymore. Remember
-that – assuming you didn't delete it in the earlier step – some of the `make`
+delete the Makefile yourself if you don't think it's needed anymore.
+
+Remember that – assuming you didn't delete it in the earlier step – some of the `make`
 commands come from `n-gage` itself and won't necessarily be in the Makefile,
 such as `make install`.
 
+You'll also want to ensure that:
+
+- You use the same node entry point of the app on `npm start`
+- You pull through any env set in the Makefile to an .rc file eg. for tests
+
 ## Migrating Your Heroku Pipeline
 
-One thing that the migration tool cannot automate is reconfiguring your Heroku
-pipeline to integrate properly with Tool Kit. You will need to ensure that your
-pipeline is connected to the project's GitHub repository, that PRs create review
-apps, and commits to main deploy from a staging app. This might be how your
-pipeline was already configured, but you should check to make sure that
-automatic deployments are still working, and try reconnecting to GitHub if they
-are not. A step-by-step guide with screenshots can be found
+The migration tool cannot automate reconfiguring your Heroku
+pipeline to integrate with Tool Kit. You will need to ensure that:
+
+- Your pipeline is connected to the project's GitHub repository
+- PRs create review apps
+- Commits to main can be deployed from a staging app
+
+This might be how your pipeline was already configured, but you should check to make sure that
+automatic deployments are still working. Try reconnecting to GitHub if they
+are not.
+
+A step-by-step guide with screenshots can be found
 [here](https://docs.google.com/document/d/1b7WlRfhiWlbDsSSGP3TllYaGMJbx9nCdAtcr8_OWEWM).
+
+## To conclude
+
+To ensure the migration has succeeded, you can run the following commands locally:
+
+`npm build` \
```

Comment:
**todo**: this is not a valid command (the others are conveniences provided by npm as aliases for `npm run start` and `npm run test` respectively)

```suggestion
`npm run build` \
```